glpi/files/etc/php/conf.d/glpi.ini

```diff
@@ -2,3 +2,10 @@
 ; Session cookies security
 session.cookie_httponly = on
 session.cookie_samesite = "Strict"
+
+; opcache settings (similar to those of glpi private cloud)
+opcache.memory_consumption=1024
+opcache.max_accelerated_files=100000
+opcache.file_update_protection=0
+opcache.fast_shutdown=1
```

Comment:
According to the PHP documentation (https://www.php.net/manual/en/opcache.configuration.php#ini.opcache.fast-shutdown):
> This directive has been removed in PHP 7.2.0. A variant of the fast shutdown sequence has been integrated into PHP and will be automatically used if possible.

```suggestion
```